configure.ac

```diff
@@ -1293,13 +1293,13 @@ AS_IF([test "x$enable_posix_cap" != "xno"],
 AC_SUBST(use_posix_cap)
 
 #
-# If the OS is linux, we can use the '--enable-linux-native-aio' option to
+# If the OS is linux, we can use the '--enable-linux-native-aio-EXPERIMENTAL' option to
 # replace the aio thread mode. Effective only on the linux system.
 #
 
 AC_MSG_CHECKING([whether to enable Linux native AIO])
-AC_ARG_ENABLE([linux-native-aio],
-  [AS_HELP_STRING([--enable-linux-native-aio], [enable native Linux AIO support @<:@default=no@:>@])],
+AC_ARG_ENABLE([linux-native-aio-EXPERIMENTAL],
```

Comment:
I'm ok with this, but it seems a little "loud" :-). Maybe just

    --enable-experimental-linux-native-aio

This will still disable this for those package managers that have been using the old configure option.